.changelog/3636.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+internal/pkg/gitdirty: Fix vcs git dirty err from showing when wpUrl is inputted without git@ prepended to url
```

Comment:
As a user of Waypoint, this release note might not be that informative. `internal/pkg/gitdirty` is as said internal and users might not know exactly what it does. 

Could we re-word this to better explain what change the users could expect? From reading the PR it seems like something to the effect of "fix an issue detecting dirty local source code when using `git@` remotes" or something?